neptune/internal/containers/disk_queue.py

```diff
@@ -83,6 +84,11 @@ def get_batch(self, size: int) -> List[T]:
     def flush(self):
         self._writer.flush()
 
+    def is_overflowing(self) -> bool:
```

Comment:
How is this used?

Reply:
Consumer is woken up before its usual sleep time has expired if there is too many operations in the queue.